BrightID/src/components/Onboarding/RecoveryFlow/RecoveryCodeScreen.tsx

```diff
@@ -380,21 +391,16 @@ const RecoveryCodeScreen = ({ route }) => {
             </Svg>
 
             <TouchableOpacity style={styles.copyContainer} onPress={copyQr}>
-              <Material
-                size={24}
-                name="content-copy"
-                color={LIGHT_BLACK}
-                style={{ width: 24, height: 24 }}
-              />
-              <Text style={styles.copyText}> {t('common.button.copy')}</Text>
+              <Copy/>
+              <Text style={styles.copyText}> {t('common.button.copyLink')}</Text>
             </TouchableOpacity>
-            {__DEV__ && (
+            {/* {__DEV__ && (
```

Comment:
This info is only displayed when running in dev mode. Is there a reason to remove it? We need it for debugging purpose.

Reply:
Added the debug info again to fix end-2-end tests.